### ISSUE TITLE
Error when rotating the phone while video is playing

### ISSUE THREAD

**Issue by danny66900** [polymorphicshade/Tubular#2]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.26.1

### Steps to reproduce the bug

1. Open any video
2. Rotate your phone 

### Expected behavior

Video should continue playing and adapt to landscape mode

### Actual behavior

App spits out an error 

### Screenshots/Screen recordings

https://streamable.com/02cpyr

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ CA
* __Content Language:__ fr-CA
* __App Language:__ fr_CA
* __Service:__ none
* __Version:__ 0.26.1
* __OS:__ Linux Android 14 - 34
<details><summary><b>Crash log </b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.schabi.newpipe.extractor.stream.StreamInfo.getUploaderName()' on a null object reference
	at org.schabi.newpipe.fragments.list.sponsorblock.SponsorBlockFragment.onCreateView(SponsorBlockFragment.java:117)
	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2995)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:523)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:261)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1374)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2841)
	at androidx.fragment.app.FragmentManager.dispatchViewCreated(FragmentManager.java:2777)
	at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3020)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:551)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:261)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1374)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2841)
	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2784)
	at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:262)
	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:478)
	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:248)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1582)
	at android.app.Activity.performStart(Activity.java:8654)
	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3847)
	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:225)
	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:205)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:177)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2468)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8248)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)

```
</details>
<hr>

### Affected Android/Custom ROM version

Android 14 CalyxOs 5.3.0

### Affected device model

Google pixel 6

### Additional information

EDIT : Works fine when sponsorblock is disabled

**Comment by polymorphicshade** [comment]
Fixed in v0.26.1 R1.

**Comment by danny66900** [comment]
> Fixed in v0.26.1 R1.

With that update I just can't open the app anymore, let me know if you want another issue 

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ CA
* __Content Language:__ fr-CA
* __App Language:__ fr_CA
* __Service:__ none
* __Version:__ 0.26.1_r1
* __OS:__ Linux Android 14 - 34
<details><summary><b>Crash log </b></summary><p>

```
io.reactivex.rxjava3.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | android.database.sqlite.SQLiteException: table `sponsorblock_whitelist` already exists (code 1 SQLITE_ERROR): , while compiling: CREATE TABLE `sponsorblock_whitelist` (`uploader` TEXT NOT NULL, PRIMARY KEY(`uploader`));
	at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:718)
	at io.reactivex.rxjava3.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:715)
	at io.reactivex.rxjava3.internal.subscribers.LambdaSubscriber.onError(LambdaSubscriber.java:79)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableObserveOn$BaseObserveOnSubscriber.checkTerminated(FlowableObserveOn.java:211)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableObserveOn$ObserveOnSubscriber.runAsync(FlowableObserveOn.java:395)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableObserveOn$BaseObserveOnSubscriber.run(FlowableObserveOn.java:178)
	at io.reactivex.rxjava3.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:123)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8248)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: android.database.sqlite.SQLiteException: table `sponsorblock_whitelist` already exists (code 1 SQLITE_ERROR): , while compiling: CREATE TABLE `sponsorblock_whitelist` (`uploader` TEXT NOT NULL, PRIMARY KEY(`uploader`));
	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1069)
	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:673)
	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:62)
	at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:34)
	at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:2088)
	at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:2010)
	at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.execSQL(FrameworkSQLiteDatabase.kt:246)
	at org.schabi.newpipe.database.Migrations$7.migrate(Migrations.java:242)
	at androidx.room.RoomOpenHelper.onUpgrade(RoomOpenHelper.kt:91)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onUpgrade(FrameworkSQLiteOpenHelper.kt:253)
	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:416)
	at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:316)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableOrReadableDatabase(FrameworkSQLiteOpenHelper.kt:232)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.innerGetDatabase(FrameworkSQLiteOpenHelper.kt:190)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getSupportDatabase(FrameworkSQLiteOpenHelper.kt:151)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.kt:104)
	at androidx.room.RoomDatabase.inTransaction(RoomDatabase.kt:638)
	at androidx.room.RoomDatabase.assertNotSuspendingTransaction(RoomDatabase.kt:457)
	at androidx.room.RoomDatabase.query(RoomDatabase.kt:486)
	at androidx.room.util.DBUtil.query(DBUtil.kt:75)
	at org.schabi.newpipe.database.feed.dao.FeedDAO_Impl$9.call(FeedDAO_Impl.java:513)
	at org.schabi.newpipe.database.feed.dao.FeedDAO_Impl$9.call(FeedDAO_Impl.java:510)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromCallable.subscribeActual(MaybeFromCallable.java:47)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5377)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableFlatMapMaybe$FlatMapMaybeSubscriber.onNext(FlowableFlatMapMaybe.java:131)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableObserveOn$ObserveOnSubscriber.runAsync(FlowableObserveOn.java:403)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableObserveOn$BaseObserveOnSubscriber.run(FlowableObserveOn.java:178)
	at io.reactivex.rxjava3.internal.schedulers.ExecutorScheduler$ExecutorWorker$BooleanRunnable.run(ExecutorScheduler.java:324)
	at io.reactivex.rxjava3.internal.schedulers.ExecutorScheduler$ExecutorWorker.runEager(ExecutorScheduler.java:289)
	at io.reactivex.rxjava3.internal.schedulers.ExecutorScheduler$ExecutorWorker.run(ExecutorScheduler.java:250)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)

```
</details>
<hr>

**Comment by ksdrummer** [comment]
Hello, I still had the issue on a fresh install of version 26.1.r2
I turned off sponsorblock and didn't experience the issue.
Now I just turned on sponsportblock again and the issue seems to be gone.
🙃
